### PR TITLE
Send button round corner and hover done in reactjs

### LINKED ISSUE #899: Round corners and Hover of Send button
I would like to add rounded corners to the send button present inside the form and its hover color.
Can I work on this issue?

**Before**
Screenshot (107)
**After**
Screenshot (108)
**On Hover**
Screenshot (109)

### PR DESCRIPTION
**Pull request by geniusbharti**
**Fixes #899** 
Send button round corners and hover effect done in reactjs version.

### CODE REVIEW

File: src/components/Footer/Footer.css [hunk 1]
```diff
@@ -119,9 +119,9 @@ form {
     width: 100%;
     margin-top: 2rem;
     padding: 3%;
-    background: #6a00bb;
-    border-bottom: 2px solid #6a00bb;
-    border-radius: 2px;
+    background: #bb86fc;
```
Comment (smaranjitghose) [inline src/components/Footer/Footer.css:122]: Declare variables for colors please with appropriate names

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.css (hunk 1 above)
Comment (geniusbharti) [reply src/components/Footer/Footer.css:122]: okay

**Comment by smaranjitghose** [comment]
The change is fine. Just update the mobile responsiveness of the footer please for all the smaller screens for a better UX

## Your PR for the React Version:

Responsiveness


## Master Branch Vesion (the hosted one):

Responsiveness_2

**Comment by smaranjitghose** [comment]
Any updates @geniusbharti ?

**Comment by smaranjitghose** [comment]
To be covered in #932

**Review by smaranjitghose — APPROVED** [review]

**Pull request merged**